scry.gemspec

```diff
@@ -21,6 +22,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "bundler"
   gem.add_development_dependency "pry-byebug"
   gem.add_development_dependency "rspec"
+  gem.add_dependency "sidekiq", "~> 4.2"
```

Comment:
ahh, I just couldn't find it in http://guides.rubygems.org/specification-reference/